app/services/services.py

```
@@ -0,0 +1,13 @@
+from fastapi.params import Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.database.repositories.currencies_repository import CurrenciesRepository
+from app.database.session import get_db
+from app.gw2.client import GW2Client
+from app.services.currencies_service import CurrenciesService
+
+
+def get_currencies_service(db: AsyncSession = Depends(get_db)) -> CurrenciesService:
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)